benchmark/1mo_benchmark.yml

```diff
@@ -47,7 +48,8 @@ data:
       subdir: OutputDir
       bmk_start: "2019-07-01T00:00:00" 
       bmk_end: "2019-08-01T00:00:00"
-      is_legacy: False
+      is_pre_13.1: False
+      is_pre_14.0: True
```

Comment:
Shouldn't is_pre_14.0 be False for dev (to be consistent with the other yaml files)?

Reply:
It looks like this version is missing my last commit to this branch.